fil-proofs-tooling/scripts/run-remote.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+CMDS=$(cat <<EOF
+
+export RUST_LOG=info
+set -e
+
+# Creates a temporary directory in which we build rust-fil-proofs and capture
+# performance metrics. The name of the directory (today's UTC seconds plus 24
```

Comment:
Wouldn't it make more sense to put the metrics with the current date and let the cleanup call remove everything from yesterday and older?

But I'm sure you've thought about that, so feel free to just leave it like that :)

Reply:
> Wouldn't it make more sense to put the metrics with the current date and let the cleanup call remove everything from yesterday and older?

The way which I've done things keeps the cleanup mechanism ignorant of any expiration policy (in this case, that the directory should be deleted one day after creation); the thing which creates the directory selects an expiration date.